server/test/app.js

```diff
@@ -1,11 +1,48 @@
+const assert = require('chai').assert;
+const expect = require('chai').expect;
+const race = require('../scripts/race.js');
 
-const request = require('supertest');
-const app = require('../app.js');
+describe('race', function() {
+  describe('calculateMotorDistance', function() {
+    it('should get the average distance traveled between the first and last packets from the 2 odometers, returned in km', function() {
+      // Arrays should be sorted newest first
+      const packetArray = [
+        {'motor0odometer': 10000, 'motor1odometer': 1000},
+        {'motor0odometer': 5000, 'motor1odometer': 2000},
+        {'motor0odometer': 5000, 'motor1odometer': 0},
+      ];
+      assert.equal(3.0, race.getDistanceTraveled(packetArray));
+    });
 
-describe('GET /random-url', () => {
```

Comment:
Just want to ensure removing this test case was intentional

Reply:
Yes, I'm not sure if it works lol